sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/SqlDelightCompiler.kt

```diff
@@ -52,13 +52,14 @@ object SqlDelightCompiler {
   ) {
     val fileIndex = SqlDelightFileIndex.getInstance(module)
     val packageName = "${fileIndex.packageName}.$implementationFolder"
-    val exposerClass = ClassName(fileIndex.packageName, "${fileIndex.className}ImplExposer")
     val outputDirectory = "${fileIndex.outputDirectory}/${packageName.replace(".", "/")}"
-    val databaseImplementationType = QueryWrapperGenerator(module, sourceFile, exposerClass).type(packageName)
-    val exposer = DatabaseExposerGenerator(databaseImplementationType, fileIndex).exposerObject()
+    val databaseImplementationType = QueryWrapperGenerator(module, sourceFile).type(packageName)
+    val exposer = DatabaseExposerGenerator(databaseImplementationType, fileIndex)
 
     FileSpec.builder(packageName, databaseImplementationType.name!!)
-        .addType(exposer)
+        // TODO: Remove these when kotlinpoet supports top level types.
```

Comment:
Hmm I don't follow?

Reply:
if i can declare the extension functions as types then i dont need to manually import, and thats merged but not yet released in kotlinpoet

Reply:
Ah the member reference. Shouldn't that comment be down below on the addStaticImport line?

Reply:
yes. i forgot to move it 🤦‍♂️